### ISSUE TITLE
Error: record does not start with '@'

### ISSUE THREAD

**Issue by dcastaneda5** [benjjneb/dada2#685]
Hello!
I am having a problem trying to plot one of my 20 reverse fastq files. Whenever I run this piece of code:
```
> plotQualityProfile(fnRs[7])
Error: record does not start with '@'
```

I verified that the number of '+' matches the number of lines divided by 4 by using the following code:
```
awk '{s++}END{print s/4}' sample7-msp1_S1_L001_R2_001.fastq
grep '^+$' sample7-msp1_S1_L001_R2_001.fastq | wc -l

```
Also, I made sure that the number of nucleotides in my reads matches the ASCII quality characters by using the following code:
`cat sample7-msp1_S1_L001_R2_001.fastq | paste - - - - | awk -F"\t" '{ if (length($2) != length($4)) print $0 }' | wc -l
`
I also looked at the plots produced in FastQC report, and it produces output. 

The tutorial is awesome and the software seems to be very useful! Thank you for that!

P.S. I am attaching the file I am having problems with.
[sample7-msp1_S1_L001_R2_001.fastq.gz](https://github.com/benjjneb/dada2/files/2887266/sample7-msp1_S1_L001_R2_001.fastq.gz)

**Comment by benjjneb** [comment]
On my end, I don't have any issue here:

```
plotQualityProfile("~/Desktop/sample7-msp1_S1_L001_R2_001.fastq.gz")
```

![rplot](https://user-images.githubusercontent.com/5797204/53142615-92f35b00-3562-11e9-90c7-5935099695d0.png)

What OS/R version are you workign with? What is `R.version`?

**Comment by dcastaneda5** [comment]
Huh, interesting... I am working with the R 3.5.2, and the macOS 10.4 Mojave. Could the OS be the problem here? I ran it again and keep getting that result...

**Comment by benjjneb** [comment]
OS 10.4? I don't know but that is pretty old these days, can you try it on a different computer?

**Comment by benjjneb** [comment]
But Mojave... oh you probably meant 10.14 which is up to date. Stilll not sure, but perhaps trying on a different machine would still be the best next thing to try.

**Comment by dcastaneda5** [comment]
I realized I sent you the seventh file thinking that the fnRs[7] would be it, but while zipping it, it was sorted in a different order. Once I looked at the file (file #15), it was corrupted. I fixed it and the graph is fine now. Thanks for the help! And apologies for the silly mistake!

**Comment by manuelzc** [comment]
Hi,
I am having the same problem. I've got my sequences in gz files dowloaded from Google Drive. I've checked them and seem to be all right, as far as I can tell. However, when I try plotQualityProfile I get an error message: Error: record does not start with '@'. I am running dada2 in windows 10, RGui ver 3.6.3. Thanks in advance for any suggestion.
[C-4-5-DI_S_R1.fastq.gz](https://github.com/benjjneb/dada2/files/5254775/C-4-5-DI_S_R1.fastq.gz)

I am attaching one of the files I am having problems with
